src/test/scala/dsptools/ResetInverterSpec.scala

```diff
@@ -0,0 +1,43 @@
+// See LICENSE for license details.
+
+package dsptools
+
+import chisel3._
+import chisel3.util.RegInit
+import dsptools.firrtltools.ResetInverter
+import firrtl._
+import org.scalatest.{FreeSpec, Matchers}
+
+class Unit extends Module with ResetInverter {
+  val io = IO(new Bundle {
+    val out = Output(UInt(32.W))
+  })
+
+  val r = RegInit(0.U)
+
+  invert(this)
+}
+
+class ResetNSpec extends FreeSpec with Matchers {
+
+  "Inverting reset needs to be done throughout module" in {
+    val optionsManager = new ExecutionOptionsManager("dsptools") with HasChiselExecutionOptions with HasFirrtlOptions {
```

Comment:
I just want to be sure I understand correctly- is the transformation we are running getting passed in via the annotation? Nothing in the test is being done to build a custom firrtl compiler? If so, that is extremely cool.